.env

```diff
@@ -0,0 +1,6 @@
+DB_USER=postgres
+DB_PASSWORD=password
+DB_NAME=postgres
+DB_HOST=localhost
+DB_PORT=15432
```

Comment:
this is for when we run a local DB with docker and set the port to foward to 15432